server/jetstream_cluster.go

```diff
@@ -1890,6 +1891,7 @@ func (js *jetStream) processStreamLeaderChange(mset *stream, isLeader bool) {
 			Sources: mset.sourcesInfo(),
 			Mirror:  mset.mirrorInfo(),
 		}
+		resp.DidCreate = true
```

Comment:
It is a bit confusing because this function (leader change) sounds like a node could be elected to be a leader for an existing stream. Say the node that is currently the leader dies, some other node may become the leader for a stream (assuming R>1). Yet, we want "DidCreate" to be true really only once, when the stream is created.

Reply:
If you look above we bail on this function before considering reporting if we are *not* the leader or have already responded.

Reply:
I see, ok then!